docs/index.rst

```diff
@@ -28,6 +28,7 @@ Vumi Go's Javascript sandbox.
    outbound.rst
    utils.rst
    test_utils.rst
+   jssandbox_tutorial/index.rst
```

Comment:
We should name this folder just `tutorial` because everything here is sandbox related. :)